NETWORK/NetworkSetInviteOnCallForInviteMenu.md

```diff
@@ -1,15 +1,15 @@
 ---
 ns: NETWORK
-aliases: ["0x66F010A4B031A331"]
+aliases: ["0x66F010A4B031A331", "_NETWORK_SET_INVITE_ON_CALL_FOR_INVITE_MENU"]
```

Comment:
Also here, I don't think this alias is necessary considering it's the same name but without a `_`.